code_samples/api/rest_api/load_content.php

```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types=1);
+
+// cURL
+
+$resource = 'https://api.example.com/api/ibexa/v2/content/objects/52';
+$curl = curl_init($resource);
+curl_setopt_array($curl, [
+    CURLOPT_HTTPHEADER => ['Accept: application/vnd.ibexa.api.ContentInfo+json'],
+    CURLOPT_HEADERFUNCTION => static function ($curl, $header) {
+        if (!empty($cleanedHeader = trim($header))) {
+            var_dump($cleanedHeader);
+        }
+
+        return strlen($header);
+    },
+    CURLOPT_RETURNTRANSFER => true,
+]);
+var_dump(json_decode(curl_exec($curl), true));
+
+// Symfony HttpClient
+
+$resource = 'https://api.example.com/api/ibexa/v2/content/objects/52';
+require 'vendor/autoload.php';
+$client = Symfony\Component\HttpClient\HttpClient::create();
+$response = $client->request('GET', $resource, [
+    'headers' => ['Accept: application/vnd.ibexa.api.ContentInfo+json'],
+]);
+var_dump($response->getStatusCode(), $response->getInfo('response_headers')[0], $response->getHeaders(false), $response->toArray(false));
```

Comment:
Why are you forcing HttpClient to not throw any exceptions? I have a feeling, it may be perceived as a good practice. To keep the example concise I'd just let the exceptions to be thrown without really catching and dealing with them.

Reply:
Because this is `var_dump` and I want to see those headers even if something went wrong. But your right. I'll try something else.

Reply:
In https://github.com/ibexa/documentation-developer/compare/616622550b7044dcbc0682f4902d4fe6df1df1a9...ab4926d330ed65c3bc7f0614143720d77e4db241 I shortened this dump to `var_dump($response->getStatusCode(), $response->getHeaders(), $response->toArray());` with exception throwing.